ui/app/pages/swaps/loading-swaps-quotes/loading-swaps-quotes.js

```diff
@@ -69,12 +69,8 @@ export default function LoadingSwapsQuotes({
   const quotesRequestCancelledEventConfig = {
     event: 'Quotes Request Cancelled',
     category: 'swaps',
-  }
-  const anonymousQuotesRequestCancelledEventConfig = {
-    event: 'Quotes Request Cancelled',
-    category: 'swaps',
     excludeMetaMetricsId: true,
```

Comment:
I'm guessing this was left here accidentally:

```suggestion
```

Reply:
There are a few more cases like this later in the PR as well

Reply:
🤦‍♂️ thanks, @Gudahtt